.github/workflows/ci.yaml

```diff
@@ -61,6 +61,9 @@ jobs:
 
   Unit-Tests:
     runs-on: ubuntu-latest
+    env:
+      NOTIFICATIONS_EMAIL_USER: no-reply@senecacollege.ca
```

Comment:
This might become redundant once https://github.com/Seneca-CDOT/starchart/pull/162 is merged.
These variable are in env.example so they will be loaded because of this change: 
<img width="400" alt="image" src="https://user-images.githubusercontent.com/67077705/218296870-245fb8db-6fd9-47bc-b46f-b71bf74f1303.png">

We could remove these later after the merge. Or do you want to implement the same change here?

Reply:
Unit Tests are not failing because of adding `env` vars in the CI file for now. I can remove them later.